### PR TITLE
feat: QA Docs Page

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary
Adds a page to document the QA process.

### Location
page under developer corner

## Structure and Tasks:

- [x] QA @ FlyByWire
- [ ] ~FlyByWire QA Process Overview~
- [x] Requirements: Github
- [x] Becoming a QA Tester
- [x] What to test (QA board)
- [x] How to get PR builds
- [x] How to test 
  - [x] Tiers
- [x] Reporting on Tests 
  - [x] Template
- [x] Roles 
- [x] Contact

### CODE REVIEW

**Comment by Valastiri** [comment]
lgtm so far

File: docs/dev-corner/qa-process.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+!!! attention ""
+    **Please do not submit reports to GitHub without having a QA Discord role.**
+
+## What to Test
+
+The FlyByWire development process provides a constant stream of new Pull Requests to be tested as each and every single code change goes through this workflow. All commits to the master branch (Development Version) are therefore code-reviewed and QA-tested.
+
+To manage this process we use the GitHub Project feature.
+
+This is a simple Kanban board where items move from the left to the rights. For QA tests the third column is relevant ("QA Team Review: Ready to Test"
```
Comment (BlueberryKing) [inline docs/dev-corner/qa-process.md]:
```suggestion
This is a simple Kanban board where items move from the left to the right. For QA tests the third column is relevant ("QA Team Review: Ready to Test")
```

File: docs/dev-corner/qa-process.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+Typically, it is ok to keep your Navigraph data installed.
+
+!!! tip "Tip: Use MSFS Addons Linker"
+    An easy way to manage add-ons in the Community folder is the [MSFS Addons Linker](https://flightsim.to/file/1572/msfs-addons-linker){target=new} tool from [flightsim.to](https://flightsim.to/).
+
+    It allows you to create presets to easily switch between different sets of installed add-ons. You can create a preset with only the "flybywire-aircraft-a320-neo" folder left in the Community folder.
+
+### Focus on the PR Test Instructions
+
+Read the test instructions of the PR carefully. Focus only on what is requested and part of this PR. Do not bloat you reports with many additional observations.
```
Comment (BlueberryKing) [inline docs/dev-corner/qa-process.md]:
```suggestion
Read the test instructions of the PR carefully. Focus only on what is requested and part of this PR. Do not bloat your reports with many additional observations.
```

File: docs/dev-corner/qa-process.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+## QA at FlyByWire
+
+FlyByWire Simulations aims to maintain professional standards as well as industry-wide best practices in order to maximize developer productivity and deliver high-quality, tested products.
+
+This is why we adopt a set of rules and practices that create a strong framework, not only around the usage of git but also surrounding code quality.
+
+Therefore, every change or addition to the code of our aircraft goes through a multi-step development and quality assurance process.
+
+This documentation describes the FlyByWire QA Process and tools in detail to make sure, we can efficiently and effectively delivery the highest quality in our products.
```
Comment (BlueberryKing) [inline docs/dev-corner/qa-process.md]:
```suggestion
This documentation describes the FlyByWire QA Process and tools in detail to make sure, we can efficiently and effectively deliver the highest quality in our products.
```

File: docs/dev-corner/qa-process.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+3. On the left side click on the bottom PR label.
+
+    ![GitHub PR Check PR](assets/qa-process/pr-check-pr.png "GitHub PR Check PR")
+
+4. Download the zipped Artifact at the bottom.
+
+    ![GitHub PR Artifact](assets/qa-process/pr-checks-artifact.png "GitHub PR Artifact")
+
+5. Unzip the file and place the "flybywire-aircraft-a320-neo" folder into you Community.
```
Comment (BlueberryKing) [inline docs/dev-corner/qa-process.md]:
```suggestion
5. Unzip the file and place the "flybywire-aircraft-a320-neo" folder into your Community.
```

File: docs/dev-corner/qa-process.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+!!! attention "PR Builds only for FlyByWire QA Team"
+    The PR builds are only meant for QA testing and not for daily use. Outside of the FlyByWire QA Team we do not provide any support or answer any questions in regards to these builds.
+
+## How to Test
+
+### QA Tiers for PRs
+
+The FlyByWire developers are required to categorize their PRs into two test tiers to indicate the complexity or importance of the PR.
+
+More complex features or features vital for the stability of the add-on will need considerably more testing then a simple change of a button text.
```
Comment (BlueberryKing) [inline docs/dev-corner/qa-process.md]:
```suggestion
More complex features or features vital for the stability of the add-on will need considerably more testing than a simple change of a button text.
```

File: docs/dev-corner/qa-process.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+- Extensive and careful testing is expected but no full flight is required.
+- Providing media is not required.
+- Of course you can always do a full flight and provide media but the test is valid without it.
+
+#### Tier 2
+
+- At least one full flight is required to test Tier 2.
+- Providing media (screenshots/videos) is a requirement especially for any issues that might arise.
+- Tests without a full flight or media are not valid and will require other tester to still do a full tier 2 test.
```
Comment (BlueberryKing) [inline docs/dev-corner/qa-process.md]:
```suggestion
- Tests without a full flight or media are not valid and will require another tester to still do a full tier 2 test.
```

**Review by BlueberryKing — CHANGES_REQUESTED** [review]
lgtm! Just a couple of typos I spotted

**Comment by frankkopp** [comment]
> lgtm! Just a couple of typos I spotted

Thanks a lot! Accepted all changes.

**Review by BlueberryKing — APPROVED** [review]

**Review by Valastiri — APPROVED** [review]

**Pull request merged**